src/common/runtime/src/global.rs

```diff
@@ -95,48 +90,38 @@ macro_rules! define_spawn {
 }
 
 impl GlobalRuntimes {
-    define_spawn!(read);
-    define_spawn!(write);
-    define_spawn!(bg);
+    define_spawn!(global);
+    define_spawn!(compact);
     define_spawn!(hb);
 
-    fn new(
-        read: Option<Runtime>,
-        write: Option<Runtime>,
-        background: Option<Runtime>,
-        heartbeat: Option<Runtime>,
-    ) -> Self {
+    fn new(global: Option<Runtime>, compact: Option<Runtime>, heartbeat: Option<Runtime>) -> Self {
         Self {
-            read_runtime: read
-                .unwrap_or_else(|| create_runtime("global-read", "read-worker", READ_WORKERS)),
-            write_runtime: write
-                .unwrap_or_else(|| create_runtime("global-write", "write-worker", WRITE_WORKERS)),
-            bg_runtime: background
-                .unwrap_or_else(|| create_runtime("global-bg", "bg-worker", BG_WORKERS)),
+            global_runtime: global
+                .unwrap_or_else(|| create_runtime("global", "global-worker", GLOBAL_WORKERS)),
+            compact_runtime: compact
+                .unwrap_or_else(|| create_runtime("compact", "compact-worker", COMPACT_WORKERS)),
             hb_runtime: heartbeat
```

Comment:
It turns out that `hb` means `heartbeat`. How about making it clear?

Reply:
I changed the runtime name. The thread name has char limits